src/store/reducers/executeQuery.ts

```diff
@@ -82,6 +83,14 @@ const executeQuery: Reducer<ExecuteQueryState, ExecuteQueryAction> = (
         }
         // 401 Unauthorized error is handled by GenericAPI
         case SEND_QUERY.FAILURE: {
+            if (isNetworkError(action.error)) {
```

Comment:
Maybe it should be any `ParseRequestError` common function? In 3 places parsing logic is the same.